docs/installation/micro-rdk-dev.md

```diff
@@ -251,6 +256,22 @@ To obtain this:
 
 {{% snippet "secret-share.md" %}}
 
+### Add your machine cloud credentials as a file in your project
+
+Navigate to the directory of the project you just created.
+Create a new <file>viam.json</file> file and paste the same `viam-server` machine cloud credentials in.
+Save the file.
+
+You can do this on the command line like this:
+
+```sh { class="command-line" data-prompt="$"}
+cd <your-path-to/your-project-directory>
+touch viam.json
+vim viam.json
+```
+
+Replace `vim` with whatever text editor you prefer.
```

Comment:
we'll leave that to the reader. If they don't know how to do this, they won't be able to develop with rust I think
```suggestion
```